src/api.rs

```diff
@@ -501,7 +514,20 @@ pub struct TransformKey {
     hashed_temp_key: HashedValue,
     public_signing_key: PublicSigningKey,
     signature: Ed25519Signature,
-    _internal_key: internal::SignedValue<internal::ReencryptionKey<Fp256>>,
+    pub(crate) _internal_key: internal::SignedValue<internal::ReencryptionKey<Fp256>>,
```

Comment:
Does this need to be `pub(crate)`?
```suggestion
    _internal_key: internal::SignedValue<internal::ReencryptionKey<Fp256>>,
```

Reply:
I thought it had to be so we could use it from the impl, but I was wrong. Thanks!